tic_tac_toe/game.py

```diff
@@ -63,22 +101,45 @@ def _check_winning_combinations(board, player):
     Returns the player (winner) of any of the winning combinations is completed
     by given player, or None otherwise.
     """
-    pass
+    winning_combinations = (
+        ((0, 0), (0, 1), (0, 2)),
+        ((1, 0), (1, 1), (1, 2)),
+        ((2, 0), (2, 1), (2, 2)),
+        ((0, 0), (1, 0), (2, 0)),
+        ((0, 1), (1, 1), (2, 1)),
+        ((0, 2), (1, 2), (2, 2)),
+        ((0, 0), (1, 1), (2, 2)),
+        ((0, 2), (1, 1), (2, 0))
+    )
+
+    if any(combination for combination in winning_combinations if _is_winning_combination(board, combination, player)):
+        return player
 
+    return None
 
 # public interface
 def start_new_game(player1, player2):
     """
     Creates and returns a new game configuration.
     """
-    pass
+    return {
+        'player1': "X",
+        'player2': "O",
```

Comment:
might be better to replace `'X` and `'O'` with `player1` and `player2` respectively. that way it's more dynamic cos players can choose their own markers.